### PR TITLE
Add Composite() to PubKey interface

### PR DESCRIPTION
**Pull request by alessio**
ref cosmos/cosmos-sdk#2019

* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [x] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 * Go API
 
+  * [\#2825](https://github.com/tendermint/tendermint/pull/2825) Add Composite() field to PubKey interface
```
Comment (melekes) [inline CHANGELOG_PENDING.md]:
```suggestion
  * \#2825 Add Composite() func to PubKey interface, which returns true if the pubkey is multisig
```

File: crypto/crypto.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ type PubKey interface {
 	Bytes() []byte
 	VerifyBytes(msg []byte, sig []byte) bool
 	Equals(PubKey) bool
+	Composite() bool
```
Comment (melekes) [inline crypto/crypto.go]: godoc?

Comment (alessio) [reply crypto/crypto.go]: Done, thanks

**Comment by cwgoes** [comment]
This could be useful anyways, but I don't think we can solve the SDK-side problem this way - see https://github.com/cosmos/cosmos-sdk/issues/2019#issuecomment-438252878.

**Comment by alessio** [comment]
This PR is the only way I can distinguish a multisig from a single-sig key.
Required to solve cosmos/cosmos-sdk#2019

Please @ValarDragon unblock.

**Pull request closed without merging**